packetbeat/protos/dns/dns.go

```diff
@@ -469,7 +469,7 @@ func addDnsToMapStr(m common.MapStr, dns *mkdns.Msg, authority bool, additional
 
 	m["answers_count"] = len(dns.Answer)
 	if len(dns.Answer) > 0 {
-		m["answers"] = rrsToMapStrs(dns.Answer)
+		m["answers"] = ansToMapStrs(dns.Answer)
```

Comment:
I agree. I just wanted to start this somewhere. 
Btw: For e.g. the "flags" fields this is already done in this way.